### PR TITLE
Fix/tao 8599/fix progress translations

### PR DESCRIPTION
**Pull request by tikhanovichA**
Depends on https://github.com/oat-sa/extension-tao-testqti/pull/1581

### CODE REVIEW

File: helpers/DeliveryHelper.php [hunk 1]
```diff
@@ -364,9 +364,19 @@ private static function adjustDeliveryExecutions($deliveryExecutions) {
         $executions = [];
         foreach ($deliveryExecutions as $cachedData) {
 
+            $progressPatern = "/(?<=\- )(.*?)(?= \d)/";
```
Comment (alexeyvy) [inline helpers/DeliveryHelper.php]: Having a test case covering this logic wouldn't be out of place

Comment (tikhanovichA) [reply helpers/DeliveryHelper.php]: It's should be done not in this PR. Also, it's the private static method from helper, and it brings a lot of complexity if we want to create a unit test exactly for that part

Comment (tikhanovichA) [reply helpers/DeliveryHelper.php]: Do you agree?

Comment (alexeyvy) [reply helpers/DeliveryHelper.php]: ok

**Pull request merged**